js/palette.js

```diff
@@ -443,15 +445,25 @@ function Palettes(canvas, refreshCanvas, stage, cellSize, refreshCanvas, trashca
             circles = showButtonHighlight(
                 palettes.buttons[name].x + r, palettes.buttons[name].y + r, r,
                 event, palettes.scale, palettes.stage);
+            
+            /*add tooltip for palette buttons*/
+            console.log("r: " + r);
+            palette_text = new createjs.Text(name, "20px Arial", "black");    
```

Comment:
@walterbender Could you please elaborate over why exactly is the "translation" needed here?